pom.xml

```diff
@@ -112,7 +112,39 @@
           <localCheckout>true</localCheckout>
         </configuration>
       </plugin>
-
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-checkstyle-plugin</artifactId>
+        <version>3.1.1</version>
+        <configuration>
+          <configLocation>checkstyle/folio_checks.xml</configLocation>
+          <encoding>UTF-8</encoding>
```

Comment:
`<linkXRef>false</linkXRef>`
should be inserted to disable this feature that is not properly configured. As suggested by the usage documentation: https://maven.apache.org/plugins/maven-checkstyle-plugin/usage.html#Checking_for_Violations_as_Part_of_the_Build

Reply:
thx. will look at that.